keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -841,7 +858,7 @@ _This section must be completed when targeting alpha to a release._
 
```

Comment:
can't comment on lines outside the diff, but I'd like to see **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?** updated with a commitment to a manual test.